--- a/CMakeLists.txt	2018-05-20 10:01:58.809531013 +0000
+++ b/CMakeLists.txt	2018-05-20 10:08:27.293815597 +0000
@@ -75,6 +75,9 @@
         SET (LIB_INSTALL_DIR "lib")
       ENDIF (CMAKE_SIZEOF_VOID_P MATCHES "8")
     ENDIF (EXISTS /etc/redhat-release OR EXISTS /etc/susehelp.d)
+    IF (EXISTS /etc/gentoo-release)
+      EXECUTE_PROCESS( COMMAND uname -m COMMAND tr -d '\n' OUTPUT_VARIABLE ARCH )
+    ENDIF(EXISTS /etc/gentoo-release)	    
   ENDIF (UNIX AND NOT APPLE)
 endif(NOT WIN32)
 
